hooks/shared/usePostRequest/usePostRequest.tsx

```diff
@@ -72,7 +70,7 @@ function usePostRequest<T, V>(
 
   const [state, dispatch] = useReducer(fetchReducer, initialState);
 
-  const postData = async () => {
+  const postData = async (postBody: T) => {
```

Comment:
Hello, rashad. As requested, I have refactored the postData function returned such that it takes in the postBody as its argument. Let me know if you have anything to say. 😁

Reply:
Looks good to me Alex! Thanks for applying the changes :D